### ISSUE TITLE
Calling bbsh without -i can still create a new build directory

### ISSUE THREAD

**Issue by darknighte** [MentorEmbedded/bbsh#1]
oe-init-build-env will create a new 'build' directory from the calling location without any parameters specified.  Currently, the wrapper script isn't smart enough to detect those conditions and abort. (e.g. no '-i' specified)

**Comment by drewmoseley** [comment]
Yes, this is definitely what I was seeing. in PR #5

**Comment by darknighte** [comment]
Fixed in PR #6

**Comment by drewmoseley** [comment]
@darknighte I don't think this was fixed in #6.  That only addressed the multi-part path names.  I still have my patch that I originally submitted.  I'll rebase now that #6 has been merged and submit it for your review.

**Comment by darknighte** [comment]
@drewmoseley I think you're correct.  Got a little trigger happy on this one.

**Comment by darknighte** [comment]
@drewmoseley, I believe my latest changes fixed the issue and added several improvements.  Please test and let me know if you still see the issue.
